### ISSUE TITLE
What should we cover in Linux?

### ISSUE THREAD

**Issue by alicerunsonfedora** [damienmsdevs/octoregister#6]
As stated from #5 , we need to discuss what should be done for Linux.

Note: _Please ensure that you are **assigned** to this issue before making a note. This is to make sure that the people who are familiar with Linux can add their comments, in addition to mentors._

**Comment by kotchet** [comment]
To start on Linux, it is better to do an overview of the base system, i.e. directories /var, /usr, /root, /bin, etc., what they are for.

**Comment by alicerunsonfedora** [comment]
There are also some other points to cover as well (at least from my experience, not in necessary order):

- Packages (What are they? What kinds of packages are there? What is a package manager? Why are packages important?)
- Customization
- Installing and configuring Linux
- Linux security (sudo, pkexec, etc.)
- Windows compatibility
- Why people choose Linux

There are probably others that I am missing, but that would be a good place to start.

**Comment by alicerunsonfedora** [comment]
I almost forgot this one too: respins, customized distros, and possibly making your _own_ distro of Linux.

**Comment by kotchet** [comment]
Making your own distro of Linux is the most important and most interesting
topic, to my opinion.

**Comment by alicerunsonfedora** [comment]
Yes, it is definitely a highlight of Linux; it seems as if all of our previous topics will eventually build up to this major one.

**Comment by alicerunsonfedora** [comment]
I think we have pretty much answered this question; @kotchet , please make the pull request and change the OVERVIEW.md document as necessary to reflect these changes.

Once the pull request is approved, I will close this issue.

**Comment by alicerunsonfedora** [comment]
Please take a look at pull request #8
